src/test/java/org/broadinstitute/hellbender/utils/IntervalUtilsUnitTest.java

```diff
@@ -1115,6 +1115,9 @@ public void testUnmergedIntervals(String unmergedIntervals) {
         merged = IntervalUtils.mergeIntervalLocations(locs, IntervalMergingRule.ALL);
         Assert.assertEquals(merged.size(), 1);
 
+        merged = IntervalUtils.mergeIntervalLocations(locs, IntervalMergingRule.NONE);
+        Assert.assertEquals(merged.size(), 2);
```

Comment:
If you remove `IntervalMergingRule.NONE` as suggested above, remove this test code too.